server/lib/configuration.js

```diff
@@ -521,6 +521,13 @@ const conf = module.exports = convict({
       threshold: 50
     }
   },
+  unblockCode: {
```

Comment:
It's super pedantic, but the mixture of camel_Case and snakeCase in this file makes me cringe. I opened https://github.com/mozilla/fxa-content-server/issues/5369 to decide on one.